recipes/libid3tag/all/test_package/conanfile.py

```diff
@@ -15,6 +15,12 @@ def requirements(self):
     def layout(self):
         cmake_layout(self)
 
+    def generate(self):
+        tc = CMakeToolchain(self)
+        if self.settings.os == "Windows" and self.dependencies[self.tested_reference_str].options.shared:
```

Comment:
Not sure about the complexity of the test, specially when this doesn't seem something that the recipe is managing.